sklearn/preprocessing.py

```diff
@@ -294,6 +296,9 @@ class StandardScaler(BaseEstimator, TransformerMixin):
     :func:`sklearn.preprocessing.scale` to perform centering and
     scaling without using the ``Transformer`` object oriented API
 
+    :class:`sklearn.preprocessing.RankScaler` to perform standardization
+    that is more robust to outliers, but slower and more memory-intensive.
```

Comment:
I think this should not be called standardization. In statistics standardization is specifically related to the Z-score transform (mean removal and standard deviation scaling), e.g.:

http://www.ats.ucla.edu/stat/stata/faq/standardize.htm

:class:`sklearn.preprocessing.RankScaler` is a feature-wise preprocessing operation that can be used as an alternative to standardization. :class:`sklearn.preprocessing.RankScaler` is more robust to outliers but slightly slower and more memory intensive.